### PR TITLE
Additional coverage for XLS dtable service

### PR DESCRIPTION
**Pull request by jomarko**
@manstis please have a look

### CODE REVIEW

File: drools-wb-screens/drools-wb-dtable-xls-editor/drools-wb-dtable-xls-editor-backend/src/test/java/org/drools/workbench/screens/dtablexls/backend/server/DecisionTableXLSServiceImplCDITest.java [hunk 1]
```diff
@@ -63,27 +66,71 @@ public void tearDown() throws Exception {
...
+        final List<ConversionMessage> messages = conversionResult.getMessages();
+        Assertions.assertThat(messages).hasSize(2);
+        SoftAssertions.assertSoftly(softly -> {
+            softly.assertThat(messages.get(0).getMessage())
+                    .isEqualTo("Unable to convert value 'Message.GOODBYE' to NUMERIC_INTEGER. Cell (F11)");
+            softly.assertThat(messages.get(1).getMessage())
+                    .startsWith("Created Guided Decision Table 'StaticFieldsTable (converted on");
+            softly.assertThat(messages.get(1).getMessage())
+                    .endsWith(").gdst'");
+            softly.assertAll();
```
Comment (triceo) [inline drools-wb-screens/drools-wb-dtable-xls-editor/drools-wb-dtable-xls-editor-backend/src/test/java/org/drools/workbench/screens/dtablexls/backend/server/DecisionTableXLSServiceImplCDITest.java]: This last line is not necessary. Happens automatically at the end of the `assertSoftly()` call.

File: drools-wb-screens/drools-wb-dtable-xls-editor/drools-wb-dtable-xls-editor-backend/src/test/java/org/drools/workbench/screens/dtablexls/backend/server/DecisionTableXLSServiceImplCDITest.java [hunk 2]
```diff
@@ -63,27 +66,71 @@ public void tearDown() throws Exception {
...
-        assertTrue(conversionResult.getMessages().get(0).getMessage().endsWith(").gdst'"));
+        final String resourcePath = "dtables/src/main/resources/guvnor/feature/dtables/FunctionAndBigDecimalTable.xls";
+        final ConversionResult conversionResult = convertResource(resourcePath);
+        final List<ConversionMessage> messages = conversionResult.getMessages();
+        Assertions.assertThat(messages).hasSize(1);
+        final String message = messages.get(0).getMessage();
+        SoftAssertions.assertSoftly(softly -> {
+            softly.assertThat(message).startsWith("Created Guided Decision Table 'FunctionAndBigDecimalTable (converted on");
+            softly.assertThat(message).endsWith(").gdst'");
+            softly.assertAll();
```
Comment (triceo) [inline drools-wb-screens/drools-wb-dtable-xls-editor/drools-wb-dtable-xls-editor-backend/src/test/java/org/drools/workbench/screens/dtablexls/backend/server/DecisionTableXLSServiceImplCDITest.java]: See below.

File: drools-wb-screens/drools-wb-dtable-xls-editor/drools-wb-dtable-xls-editor-backend/src/test/java/org/drools/workbench/screens/dtablexls/backend/server/DecisionTableXLSServiceImplCDITest.java [hunk 3]
```diff
@@ -63,27 +66,71 @@ public void tearDown() throws Exception {
...
+    @Ignore("Ignored due to RHDM-216")
+    public void testFromAccumulate() throws Exception {
+        final String resourcePath = "forest/src/main/resources/com/redhat/sample/ForestHealth.xls";
+        final List<ValidationMessage> messages = validateResource(resourcePath);
+        Assertions.assertThat(messages).hasSize(0);
+    }
+
+    private ConversionResult convertResource(final String resource) throws Exception {
+        final Path resourcePath = getPath(resource);
+
```
Comment (triceo) [inline drools-wb-screens/drools-wb-dtable-xls-editor/drools-wb-dtable-xls-editor-backend/src/test/java/org/drools/workbench/screens/dtablexls/backend/server/DecisionTableXLSServiceImplCDITest.java]: Any reason for the blank line?

File: drools-wb-screens/drools-wb-dtable-xls-editor/drools-wb-dtable-xls-editor-backend/src/test/java/org/drools/workbench/screens/dtablexls/backend/server/DecisionTableXLSServiceImplCDITest.java [hunk 4]
```diff
@@ -63,27 +66,71 @@ public void tearDown() throws Exception {
...
+        return xlsService.convert(resourcePath);
     }
 
-    private void convertResource(final String resource) throws Exception {
+    private List<ValidationMessage> validateResource(final String resource) throws Exception {
+        final Path resourcePath = getPath(resource);
+
```
Comment (triceo) [inline drools-wb-screens/drools-wb-dtable-xls-editor/drools-wb-dtable-xls-editor-backend/src/test/java/org/drools/workbench/screens/dtablexls/backend/server/DecisionTableXLSServiceImplCDITest.java]: Dtto.

**Comment by jomarko** [comment]
@triceo thanks, I will update in a minute

**Comment by jomarko** [comment]
@triceo comments incorporated now.

**Pull request merged**